src/components/SubmitResponsePage.jsx

```diff
@@ -2,8 +2,10 @@ import React from 'react';
 import FormContainer from './FormContainer';
 import { Formik, Form } from 'formik';
 import { HasResponseErrors } from '../utilities/CitysourcedResponseHelpers';
+import ButtonDisplay from "./buttonDisplay";
 
 const SubmitResponse = (props) => {
+	console.log(props);
```

Comment:
please remove any logging. this may have been me, and i apologize for that.

Reply:
this was removed